### ISSUE TITLE
False positive on Readability.LargeNumbers

### ISSUE THREAD

**Issue by ejpcmac** [rrrene/credo#560]
### Precheck

It looks like #325 but is still around and in some really specific case.

### Environment

* Credo version (`mix credo -v`): 0.9.3
* Erlang/Elixir version (`elixir -v`): Erlang/OTP 21.0 / Elixir 1.6.6
* Operating system: Fedora Linux

### What were you trying to do?

I am trying to check this code with Credo:

```elixir
defmodule ExampleTest do
  use ExUnit.Case
  use ExUnitProperties

  property "make credo warn" do
    check all integer <- integer(-10_000..-1) do
      assert is_integer(integer)
    end
  end
end
```

### Expected outcome

There should be no warnings.

### Actual outcome

Credo warns about `10_000` being written without underscores.

If I set the range to `-10_000..0`, there is no problem anymore.

**Comment by rrrene** [comment]
@ejpcmac Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by ejpcmac** [comment]
@rrrene Thank you! I’ll try this on Monday since my ready-to-try sample is at work. I’ve set a reminder.

**Comment by ejpcmac** [comment]
@rrrene It’s good! Thank you :smile: :tada:
